### PR TITLE
fix: metrics stored_index_files_size is never cleared

### PR DESCRIPTION
**Pull request by jaime0815**
issue: ##36159
pr: #36160

### CODE REVIEW

File: internal/datacoord/meta.go [hunk 1]
```diff
@@ -450,12 +450,13 @@ func (m *meta) GetQuotaInfo() *metricsinfo.DataCoordQuotaMetrics {
 	return info
 }
 
-// GetCollectionIndexFilesSize returns the total index files size of all segment for each collection.
-func (m *meta) GetCollectionIndexFilesSize() uint64 {
+// SetStoredIndexFileSizeMetric returns the total index files size of all segment for each collection.
+func (m *meta) SetStoredIndexFileSizeMetric() uint64 {
```
Comment (czs007) [inline internal/datacoord/meta.go:454]: no need to rename as SetXXX? since it returns a value.

Comment (jaime0815) [reply internal/datacoord/meta.go:454]: The primary function is to set values to metrics, so rename it from a 'get' to a 'set' method. The return value is intended solely for testing.

**Pull request merged**